x/ccv/consumer/keeper/validators.go

```diff
@@ -104,23 +102,29 @@ func (k Keeper) ValidatorByConsAddr(sdk.Context, sdk.ConsAddress) stakingtypes.V
 
 // Slash queues a slashing request for the the provider chain
 // All queued slashing requests will be cleared in EndBlock
-func (k Keeper) Slash(ctx sdk.Context, addr sdk.ConsAddress, infractionHeight, power int64, slashFactor sdk.Dec, infraction stakingtypes.InfractionType) {
-	if infraction == stakingtypes.InfractionEmpty {
+func (k Keeper) SlashForked(
+	ctx sdk.Context,
+	addr sdk.ConsAddress,
+	infractionHeight, power int64,
+	slashFactor sdk.Dec,
+	infraction stakingtypes.Infraction,
+) {
+	if infraction == stakingtypes.Infraction_INFRACTION_UNSPECIFIED {
 		return
 	}
 
 	// If this is a previously standalone chain and infraction happened before the changeover was completed,
 	// slash only on the standalone staking keeper.
 	if k.IsPrevStandaloneChain() && infractionHeight < k.FirstConsumerHeight(ctx) {
-		k.standaloneStakingKeeper.Slash(ctx, addr, infractionHeight, power, slashFactor, stakingtypes.InfractionEmpty)
+		k.standaloneStakingKeeper.Slash(ctx, addr, infractionHeight, power, slashFactor)
```

Comment:
@sontrinh16, @pysel: I think we need to review upstream changes, they may eliminate the need for SlashForked.  Also, I think that we should update godocs here, but I don't know what the difference between SlashForked and Slash is right now.  Thanks!

Reply:
oh @vuong177 is working on all the slash stuff right now

Reply:
just need to wait for him to push